wls-wahlvorbereitung-service/src/main/java/de/muenchen/oss/wahllokalsystem/wahlvorbereitungservice/rest/waehlerverzeichnis/WaehlerverzeichnisController.java

```diff
@@ -0,0 +1,47 @@
+package de.muenchen.oss.wahllokalsystem.wahlvorbereitungservice.rest.waehlerverzeichnis;
+
+import de.muenchen.oss.wahllokalsystem.wahlvorbereitungservice.rest.AbstractController;
+import de.muenchen.oss.wahllokalsystem.wahlvorbereitungservice.service.waehlerverzeichnis.WaehlerverzeichnisService;
+import de.muenchen.oss.wahllokalsystem.wls.common.security.domain.BezirkIDUndWaehlerverzeichnisNummer;
+import io.swagger.v3.oas.annotations.Operation;
+import lombok.RequiredArgsConstructor;
+import lombok.val;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/businessActions/waehlerverzeichnis")
+@RequiredArgsConstructor
+public class WaehlerverzeichnisController extends AbstractController {
+
+    private final WaehlerverzeichnisDTOMapper waehlerverzeichnisDTOMapper;
+
+    private final WaehlerverzeichnisService waehlerverzeichnisService;
+
+    @PostMapping("{wahlbezirkID}/{wvzNummer}")
+    @ResponseStatus(HttpStatus.CREATED)
+    @Operation(
+            description = "Speichern der Angaben über das Wählerverzeichnis des Urnenwahllokals {wahlbezirkID}. Wird je aktivem Wählerverzeichnis einmal gespeicehrt und per wvzNummer dem jeweiligen zugeordnet."
```

Comment:
```suggestion
            description = "Speichern der Angaben über das Wählerverzeichnis des Urnenwahllokals {wahlbezirkID}. Wird je aktivem Wählerverzeichnis einmal gespeichert und per wvzNummer dem jeweiligen zugeordnet."
```

Reply:
fixed in 7a1e80472cbf200b354e66e955e76c61e944b7ed